pymeasure/instruments/lakeshore/lakeshore211.py

```diff
@@ -0,0 +1,205 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+
+from pymeasure.instruments import Instrument
+from pyvisa.constants import Parity
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class LakeShore211(Instrument):
+    """ Represents the Lake Shore 211 Temperature Monitor and provides
+    a high-level interface for interacting with the instrument.
+
+    Untested properties and methods will be noted in their docstrings.
+
+    .. code-block:: python
+
+        controller = LakeShore211("GPIB::1")
+
+        print(controller.temperature_celsius)     # Print the sensor temperature in celsius
+
+    """
+    alarm_keys = ['on', 'high_value', 'low_value', 'deadband', 'latch']
+
+    def __init__(self, adapter, name="Lake Shore 211 Temperature Monitor", **kwargs):
+        kwargs.setdefault('data_bits', 7)
+        kwargs.setdefault('parity', Parity.odd)
+        super().__init__(
+            adapter,
+            name,
+            **kwargs
+        )
+
+    analog = Instrument.control(
+        "ANALOG?", "ANALOG %d,%d",
+        """
+        Set the analog mode and analog range.
```

Comment:
The "keywords" for `control`, `measurement`, and `setting` are "control", "get" or "measure", and "set:
```suggestion
        Control the analog mode and analog range.
```

Please change the other property factories accordingly.

Reply:
Updated docstrings to nomenclature